OpenRA.Mods.Common/Widgets/Logic/ModBrowserLogic.cs

```diff
@@ -148,10 +155,29 @@ void SelectMod(ModMetadata mod)
 			var selectedIndex = Array.IndexOf(allMods, mod);
 			if (selectedIndex - modOffset > 4)
 				modOffset = selectedIndex - 4;
+
+			loadButton.Text = modInstallStatus[mod] ? "Load Mod" : "Install Assets";
```

Comment:
My preference is to change the dialog title to "Install Assets", which then matches "Install Music".